assets/airports/ksdf.json

```diff
@@ -53,9 +53,9 @@
         }
     ],
     "fixes": {
-        "_RWY11294GADSN268": ["N38.18880", "W85.77804"],
-        "_RWY17R348RRROE333": ["N38.21339", "W85.75886"],
-        "_RWY35L168LEENE183": ["N38.14838", "W85.73555"],
+        "_RWY29294GADSN268": ["N38.18880", "W85.77804"],
+        "_RWY35L348RRROE333": ["N38.21339", "W85.75886"],
+        "_RWY17R168LEENE183": ["N38.14838", "W85.73555"],
```

Comment:
Same as with ATL, these can omit the headings from the runway whenever the heading is the runway's heading: `_RWY17RLEENE183`.

Reply:
done